ico.sol

```diff
@@ -74,9 +74,10 @@ contract ico is safeMath, owned {
         } else {
             startBlock = block.number;
         }
-        icoLevels.push(icoLevels_s(safeMul(safeAdd(startBlock, oneSegment), 1), 0));
-        icoLevels.push(icoLevels_s( startBlock, 3));
-        icoDelay = safeMul(safeAdd(startBlock, oneSegment), 2);
+        
+        icoLevels.push(icoLevels_s(safeAdd(startBlock, safeMul(oneSegment, 1)), 0));
```

Comment:
What is the reason of having `safeMul(oneSegment, 1)`? You can just push `oneSegment` without multiplying it by 1.

`oneSegment * 1` = `oneSegment`